packages/documentation/src/content/docs/integration/requirements/peers.mdx

```diff
@@ -0,0 +1,287 @@
+- Exchange `auth` tokens for the connection endpoint.
+- Agree on a settlement mechanism (a means of paying one another for the successful forwarding and delivery of packets), which is outside the scope of Interledger and Rafiki.
+
+Optionally, you may also deposit an initial liquidity for your peer.
+
+:::note
+Peering is not required unless you want to participate in transactions with another ASE running Rafiki. For foundational peering concepts, refer to [Peers](/overview/concepts/interledger/#peers).
+:::
+
+## Setting up peering in Rafiki
+
+The basic workflow of setting up a peering relationship starts with adding the agreed upon asset, then adding a peer, and then creating a wallet address from where you can send and receive payments. 
+
+### Add an asset
+
+As mentioned in the prerequisites, an asset must be added to your Rafiki instance before creating a peering relationship. To learn how to add an asset, refer to [Assets](/integration/requirements/assets/).
+
+### Add a peer
+
+After adding an asset, add a peer using the `createPeer` GraphQL mutation.
+
+#### `createPeer`
+
+<CodeBlock title="Example createPeer Mutation">
+
+```graphql
+mutation CreatePeer($input: CreatePeerInput!) {
+  createPeer(input: $input) {
+    code
+    success
+    message
+    peer {
+      id
+      asset {
+        code
+        scale
+      }
+      staticIlpAddress
+      name
+    }
+  }
+}
+```
+
+</CodeBlock>
+
+##### Example
+
+<CodeBlock title='Example JSON request'>
+
+```json
+{
+  "input": {
+	"staticIlpAddress": "g.othergreatwallet",
+	"name": "The Other Great Wallet"
+	"http": {
+  		"incoming": {"authTokens": ["mytoken"]},
+  		"outgoing": {"endpoint": "ilp.othergreatwallet.com", "authToken": "theirtoken"}
+	  },
+	"assetId": "INSERT_ASSET_ID",
+	"initialLiquidity": <optionally, an intial amount of liquity to provision. Liquidity can also be deposited via the `DepositPeerLiquidity`>
+  }
+}
+```
+
+</CodeBlock>
+
+<div class="overflow-table">
+
+| Variable                   | Description                                                                                                                 | Required |
+| -------------------------- | --------------------------------------------------------------------------------------------------------------------------- | -------- |
+| `assetID`                  | The ID of the asset that you and your peer will use to ultimately settle your net obligations outside of Interledger.       | Y        |
+| `staticILPaddress`         | Your peer’s static ILP address (the example uses `g.othergreatwallet`)                                                      | Y        |
+| `name`                     | Your peer’s name (the example uses “The Other Great Wallet”)                                                                | Y        |
+| `http.incoming.authTokens` | The token your peer will use to present and connect to and send packets to your Rafiki instance.                            | Y        |
```

Comment:
Can you check that the descriptions for incoming.authTokens and outgoing.authtoken are correct? The info on the [Backend service page](https://rafiki.dev/integration/services/backend-service/#interledger-connector) makes me question what's here. Also, is incoming plural (authTokens) and outgoing singular? Should outgoing be authToken?

Reply:
This is correct. HttpIncoming accepts an array of auth tokens while HttpOutgoing accepts a single token.